crates/astria-core/CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `CheckedTransaction` from this [#2142](https://github.com/astriaorg/astria/pull/2142).
 - Rename ABCI error code 10 from `TRANSACTION_FAILED` to
   `TRANSACTION_FAILED_EXECUTION` [#2142](https://github.com/astriaorg/astria/pull/2142).
+- Change `CurrencyPairsChange` action to hold a `BTreeSet` rather than a `Vec`
```

Comment:
```suggestion
- Change `CurrencyPairsChange` action to hold a `IndexSet` rather than a `Vec`
```

Reply:
Fixed in [97a90d8](https://github.com/astriaorg/astria/pull/2171/commits/97a90d8e0738cea54d6556314fd276af53300510).